frontend/src/scenes/App.tsx

```diff
@@ -143,10 +142,10 @@ function AppScene(): JSX.Element | null {
 
     if (!user) {
         return sceneConfig?.onlyUnauthenticated || sceneConfig?.allowUnauthenticated ? (
-            <Layout style={{ minHeight: '100vh' }}>
```

Comment:
Wonder if that 100vh was important... Took a quick look and it doesn't seem to be 😅

Reply:
Weirdly it seems to have been really irrelevant, unless there is some edge cases somewhere where the whole page breaks and Windows shows a BSOD without `min-height: 100vh` 🤔